po/fr.po

```diff
@@ -8546,8 +8546,8 @@ msgstr ""
 "ne jamais toucher sauf si vous savez ce que vous faites."
 
 #: ../src/iop/filmicrgb.c:192
-msgid "filmic rgb"
-msgstr "filmique rvb"
+msgid "filmic RGB"
```

Comment:
You can't change this string as it is the one in the code. This is the "key" that dt will use at runtime to find the translation. Changing that breaks the translation.